### PR TITLE
OpenEmbedded mapping additions to rosdep for Melodic and vario…

### PR DESCRIPTION
**Pull request by herb-kuta-lge**

### CODE REVIEW

**Review by tfoote — APPROVED** [review]

**Comment by herb-kuta-lge** [comment]
@ros/meta-ros-maintainers < Do I need to rebase this before you will merge it?

**Comment by herb-kuta-lge** [comment]
@ros/rosdeputies < Do I need to rebase this before you will merge it?

**Comment by nuclearsandwich** [comment]
> @ros/rosdeputies < Do I need to rebase this before you will merge it?

The rebase isn't required. This pull request was simply waiting on a second review as we want two for rosdep DB pull requests.

**Review by nuclearsandwich — APPROVED** [review]

**Pull request merged**
